ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml

```diff
@@ -924,6 +924,15 @@ tests:
       TEST_TYPE: conformance-parallel
     workflow: openshift-e2e-powervs-ipi
   timeout: 6h0m0s
+- as: fips-payload-scan
+  steps:
```

Comment:
I think you want this to be a `periodic` job...
You can add the following to enable a periodic:
```
interval: 168h
```